src/features/Application/validationSchema.ts

```diff
@@ -1,80 +1,262 @@
 import { array, boolean, mixed, number, object, string } from 'yup';
 
-const getValidationSchema = (isCreate: boolean) => {
+const getValidationSchema = (isCreate: boolean, pageNumber: number) => {
   const resumeSchema = isCreate
     ? mixed().required('A resume is required')
     : mixed();
-  return object().shape({
-    application: object().shape({
-      general: object().shape({
-        school: string()
-          .min(1, 'Select a school')
-          .required('Required'),
-        degree: string().required('Required'),
-        fieldOfStudy: string().required('Required'),
-        graduationYear: number()
-          .required('Required')
-          .min(2018)
-          .max(2025),
-        jobInterest: string().required('Required'),
-        URL: object().shape({
-          resume: resumeSchema
-            .test(
-              'fileSize',
-              'File too large (<4MB only)',
-              (value) => !value || value.size <= 4000000 // 4MB
-            )
-            .test(
-              'fileFormat',
-              'Unsupported Format (PDF only)',
-              (value) => !value || value.type === 'application/pdf'
-            ),
-          github: string().url('Must be a valid url'),
-          dribbble: string().url('Must be a valid url'),
-          linkedIn: string().url('Must be a valid url'),
-          personal: string().url('Must be a valid url'),
-          other: string().url('Must be a valid url'),
+  switch (pageNumber) {
+    case 1:
+      return object().shape({
+        hacker: object().shape({
+          application: object().shape({
+            general: object().shape({
+              school: string()
+                .min(1, 'Select a school')
+                .required('Required'),
+              degree: string().required('Required'),
+              fieldOfStudy: string().required('Required'),
+              graduationYear: number()
+                .required('Required')
+                .min(2018)
```

Comment:
Should these also be abstracted to the consts file?

Also, 2018 should be updated to 2019.